RELEASE_NOTES

```diff
@@ -3,6 +3,47 @@ For more detailed information, please see the git log.
 
 These release notes can also be consulted at http://easybuild.readthedocs.org/en/latest/Release_notes.html.
 
+v2.1.0 (April 28th 2015)
+------------------------
+
+feature + bugfix release
+- bump required vsc-base version to 2.2.0
+- add support for only (re)generating module files: --module-only (#1018)
+    - module naming scheme API is enhanced to include det_install_subdir method
+    - FIXME docs see
+- add support for generating module files in Lua syntax (note: required Lmod as modules tool) (#1060, #1255, #1256)
+    - see --module-syntax configuration option
+- deprecate log.error in favor of raising EasyBuildError exception (#1218)
+    - see http://easybuild.readthedocs.org/en/latest/Deprecated-functionality.html#depr-error-reporting
+- add support for using external modules as dependencies, and to provide metadata for external modules (#1230, #1265)
+    - see http://easybuild.readthedocs.org/en/latest/Using_external_modules.html
+- add experimental support for Cray toolchains on top of PrgEnv modules: CrayGNU, CrayIntel, CrayCCE (#1234)
+    - FIXME see (wiki)
```

Comment:
need to put together with wiki page on the experimental Cray support, which lists what is supported now, how it works, and what the open questions are

since this is experimental, it shouldn't go to readthedocs yet